shell/platform/darwin/ios/framework/Source/FlutterViewController.mm

```diff
@@ -1200,9 +1200,10 @@ - (void)startKeyBoardAnimation:(NSTimeInterval)duration {
       }
       completion:^(BOOL finished) {
```

Comment:
`finished` may be false here because the `keyboardAnimationView` with the animation was removed from the superView before the animation was completed? Not sure though.